code/_onclick/hud/screen_objects.dm

```diff
@@ -175,7 +175,7 @@
 	var/image/item_overlay = image(holding)
 	item_overlay.alpha = 92
 
-	if(!user.can_equip(holding, slot_id, TRUE))
+	if(!user.can_equip(holding, slot_id, disable_warning = TRUE, bypass_equip_delay_self = TRUE)) //NSV13, fixes hover triggering of equip delay
```

Comment:
```suggestion
	if(!user.can_equip(holding, slot_id, TRUE, TRUE))
```
Please port this fix upstream as well and then remove the //NSV13 tag

Reply:
i'll do that, and i'll also try to get this whole thing merged there.